homeassistant/components/unifi/sensor.py

```diff
@@ -415,6 +468,104 @@ class UnifiSensorEntityDescription(
         unique_id_fn=lambda hub, obj_id: f"memory_utilization-{obj_id}",
         value_fn=lambda hub, device: device.system_stats[1],
     ),
+    UnifiSensorEntityDescription[Devices, Device](
```

Comment:
@Kane610 It's pretty much duplicate work here. 
Was considering making a def that return the three sensors. 
Then we could at the end do something like `) + make_latency_sensors("WAN") + make_latency_sensors("WAN2")` etc.

Reply:
I was not possible, at least of that I tried, to get use methods that had `wan: str` as parameter. 

But I think we can still make dynamic methods.